server/models/instance.py

```diff
@@ -297,11 +306,16 @@ def finalizeInstance(event):
             })
             if "sessionId" in service:
                 instance["sessionId"] = ObjectId(service["sessionId"])
+
+            notify_event([str(tale["creatorId"])], "wt_instance_running", "instance",
```

Comment:
Why notify Tale creator and not instance creator?

Reply:
Good question. I didn't consider the instance creator, which obviously makes more sense.